hist/hist/inc/TF1.h

```diff
@@ -325,7 +394,7 @@ class TF1 : public TNamed, public TAttLine, public TAttFill, public TAttMarker {
       TNamed(name, name), TAttLine(), TAttFill(), TAttMarker(),
       fXmin(xmin), fXmax(xmax),
       fNpar(npar), fNdim(ndim),
-      fNpx(100), fType(1),
+      fNpx(100), fType(3),
```

Comment:
Yes... I wanted to change the minimum now, but I plan to take a look at TF1 and improve several things after this.